ProviderFactory/AlgoliaFactory.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the BazingaGeocoderBundle package.
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @license    MIT License
+ */
+
+namespace Bazinga\GeocoderBundle\ProviderFactory;
+
+use Geocoder\Provider\AlgoliaPlaces\AlgoliaPlaces;
+use Geocoder\Provider\Provider;
+use Http\Discovery\HttpClientDiscovery;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+final class AlgoliaFactory extends AbstractFactory
+{
+
```

Comment:
Unnecessary linebreak

Reply:
My common sense and eye strain wouldn't agree :D but ok, done.